samples/ImageBadgeSample.js

```diff
@@ -5,25 +5,37 @@ enyo.kind({
 	components: [
 		{kind: "moon.Scroller", fit: true, components: [
 			{kind: "moon.Divider", content: "Image Badges:"},
-			{kind: "moon.Image", src: "http://placehold.it/340x360&text=Image+One", alt: "Image One", components: [
+			{kind: "moon.Image", src: {
+				"hd" : "http://placehold.it/228x240&text=Image+One",
+				"fhd": "http://placehold.it/342x360&text=Image+One"
+			}, alt: "Image One", components: [
 				{kind: "moon.Icon", icon: "check"},
 				{kind: "moon.Icon", icon: "closex"},
 				{kind: "moon.Icon", icon: "drawer", classes: "float-right"}
 			]},
-			{kind: "moon.Image", src: "http://placehold.it/180x240&text=Image+Two", alt: "Image Two", components: [
+			{kind: "moon.Image", src: {
+				"hd" : "http://placehold.it/120x160&text=Image+Two",
+				"fhd": "http://placehold.it/180x240&text=Image+Two"
+			}, alt: "Image Two", components: [
 				{kind: "moon.Icon", icon: "closex"}
 			]},
 			
 			{kind: "moon.Divider", classes: "image-badge-sample-divider", content: "Image Badges - Show on Spotlight:"},
 			{kind: "moon.Item", components: [
-				{kind: "moon.Image", src: "http://placehold.it/340x360&text=Image+One", alt: "Image One", showBadgesOnSpotlight: true, components: [
+				{kind: "moon.Image", src: {
+					"hd" : "http://placehold.it/228x240&text=Image+One",
+					"fhd": "http://placehold.it/342x360&text=Image+One"
+				}, alt: "Image One", showBadgesOnSpotlight: true, components: [
 					{kind: "moon.Icon", icon: "check"},
 					{kind: "moon.Icon", icon: "closex"},
 					{kind: "moon.Icon", icon: "drawer", classes: "float-right"}
 				]}
 			]},
 			{kind: "moon.Item", components: [
-				{kind: "moon.Image", src: "http://placehold.it/180x240&text=Image+Two", alt: "Image Two", showBadgesOnSpotlight: true, components: [
+				{kind: "moon.Image", src: {
```

Comment:
@aarontam I think we can remain at least one sample to use string in src property instead of array